src/pages/vendor/Addons.tsx

```
@@ -4,19 +4,25 @@ import { useTranslation } from 'react-i18next';
 import { toast } from 'sonner';
 import { useParams } from 'react-router-dom';
 import Select from 'react-select';
+import { useMutation, useQuery } from '@apollo/client';
+import { CREATE_OPTION_SET, GET_OPTIONSET_MENU, UPDATE_OPTION_SET, DELETE_OPTIONSET } from '../../lib/graphql/queries/optionset';
+import { useCurrency } from '../../contexts/CurrencyContext';
 
 // Types
 interface Dish {
-  id: string;
+  foodId?: String,
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)